MAINTAINERS-example

```diff
@@ -1,33 +1,23 @@
-# This project, topsikrit, has top-secret awesome sauce.
-#
-# Btw, note that this comment was human-edited and will be preserved
-# even if you add/remove maintainers using the gem. Nifty!
-
-maintainers: {
-    version: 1
-    issues: "https://github.com/csprojects/topsikrit/issues"
-    # This is a very interesting comment. It will also be preserved.
-    roles: {
-        core: {
-            "ada": {},
-            "turing": {},
-        },
-        subsystem: {
-            "gracehopper": { subsystem: "ENIAC" },
-        },
+{
+  "version": 1,
+  "maintained": false,
+  "people": [
+    {
+      "github": "gracehopper",
+      "email": "admiral.grace.hopper@usnavy.gov",
+      "name": "Grace Hopper",
+      "comment": "Maintains ENIAC subsystem"
+    },
+    {
+      "github": "turing",
+      "email": "at@cambridge.edu",
+      "name": "Alan Turing",
+      "comment": "Passed his own test"
+    },
+    {
+      "github": "ada",
+      "email": "ada@example.com",
```

Comment:
Kind of. From a quick scan of the email field of various commits, while most people include genuine email addresses, some people prefer to put in bogus emails, e.g. with `@users.noreply.github.com` or `@github` or `@980ebf18-57e1-0310-9a29-db15c13687c0`. 

I agree that the email and name fields should be optional. I'll push an update where only the `github` field is required.